app/api/v1/endpoints/telegram.py

```
@@ -564,15 +564,27 @@ async def _verify_telegram_webhook(request: Request) -> bool:
+        is_valid = signature == webhook_secret
+        
+        if not is_valid:
+            logger.warning("Webhook verification failed", 
+                         expected_secret=webhook_secret[:8] + "...", 
+                         received_signature=signature[:8] + "..." if signature else "None")
+        else:
+            logger.info("Webhook verification passed")
+            
+        return is_valid
         
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)